float32 random_num
---
bool less_than
